core/node/eth_sender/src/aggregator.rs

```diff
@@ -133,7 +141,9 @@ impl Aggregator {
             operate_4844_mode,
             pubdata_da,
             commitment_mode,
-        }
+            priority_merkle_tree: OnceCell::new(),
```

Comment:
Nit: Since initialization in `get_or_init_tree` has exclusive access to `self`, won't `Option<_>` suffice here instead of `OnceCell`?

Reply:
It would, I thought using OnceCell would be more representative of the meaning, but okay, I will update

Reply:
`OnceCell` has interior mutability / synchronization because in the intended use case, it can be attempted to be initialized concurrently from multiple threads. `Option` is thus cheaper to use (although this is obviously not a large concern in this case), and has slightly simpler API.